command/maintenance_window_delete.go

```diff
@@ -23,5 +32,27 @@ func (c *MaintenanceWindowDelete) Synopsis() string {
 }
 
 func (c *MaintenanceWindowDelete) Run(args []string) int {
+	var mwID string
+	flags := c.Meta.FlagSet("maintenance-window delete")
+	flags.Usage = func() { fmt.Println(c.Help())}
+	flags.StringVar(&mwID, "id", "", "Maintenance window ID")
+
+	if err := flags.Parse(args); err != nil {
+		log.Error(err)
+		return -1
+	}
+	if err := c.Meta.Setup(); err != nil {
+		log.Error(err)
+		return -1
+	}
+	if mwID == "" {
+		log.Error("You must provide a maintenance window ID")
+		return -1
+	}
+	client := c.Meta.Client()
+	if err := client.DeleteMaintenanceWindowWithContext(context.Background(), mwID); err != nil {
+		log.Error(err)
+		return -1
+	}
```

Comment:
Slight style nit, could you space these blocks out into logical groupings with a newline in between?

```go
	if err := c.Meta.Setup(); err != nil {
		log.Error(err)
		return -1
	}

	if mwID == "" {
		log.Error("You must provide a maintenance window ID")
		return -1
	}

	client := c.Meta.Client()
	if err := client.DeleteMaintenanceWindowWithContext(context.Background(), mwID); err != nil {
		log.Error(err)
		return -1
	}

	return 0
```